app/(dashboard)/settings/tools/apiForm.tsx

```
@@ -10,9 +10,10 @@ import { api } from "@/trpc/react";
 
 type ApiFormProps = {
   onCancel: () => void;
+  "data-testid"?: string;
 };
 
-const ApiForm = ({ onCancel }: ApiFormProps) => {
+const ApiForm = ({ onCancel}: ApiFormProps) => {
```

ESLint

[error] Insert `·`

(prettier/prettier)